src/base/bittorrent/tracker.h

```diff
@@ -51,28 +51,33 @@ namespace BitTorrent
 {
     struct Peer
     {
-        QString ip;
-        QString peerId;
-        int port;
-
         bool operator!=(const Peer &other) const;
         bool operator==(const Peer &other) const;
-        QString uid() const;
-        libtorrent::entry toEntry(bool noPeerId) const;
+
+        // ip retrieved from connection
+        QString m_ip;
```

Comment:
Fix coding style.
`m_` prefix is used for **private** fields only.

Also keep the fields together and methods (including operators) together.